inline-lexer.js

```diff
@@ -112,6 +112,29 @@ InlineLexer.prototype.next = function next(type, text, scanner) {
             }
             this.accumulator += c;
             this.type = 'alphanum';
+        } else if (scanner.leader === '!' && (c === "'" || c === '"')) {
```

Comment:
Let’s reserve double-quote for something else maybe. We can use just single-quote for now.